docs/src/quickstart/install.rst

```diff
@@ -15,8 +15,9 @@ according to the system used:
 
  - **Linux** The GNU C Compiler (gcc) is usually present, or easily
    available through the package system. On Ubuntu or Debian, for
-   instance, the command ``sudo apt-get install build-essential`` will
-   fetch everything you need.
+   instance, the command
+   ``sudo apt-get install build-essential python3-dev`` will fetch
```

Comment:
It's probably a useful hint but I don't quite like how it's phrased because it reads to me like python-dev contains GCC

Reply:
You're absolutely right! Sorry for being so careless in proposing this change. I reworded the text so that it (hopefully) makes more sense.